experimental/generation/generator/packages/library/src/dialogGenerator.ts

```diff
@@ -232,7 +232,12 @@ function addPrefixToImports(template: string, scope: any): string {
 }
 
 function addPrefix(prefix: string, name: string): string {
-    return `${prefix}-${name}`
+    let dir = name.lastIndexOf('/')
```

Comment:
Consider using path methods to parse out the name with a directory.  Do you ever have \\ as the separator?

Reply:
This is not a path and basename doesn't do the right thing if a plain name.

---
In reply to: [491180765](https://github.com/microsoft/BotBuilder-Samples/pull/2750#discussion_r491180765) [](ancestors = 491180765)